### ISSUE TITLE
Add website links and redirects

### ISSUE THREAD

**Issue by karaxgg** [openmultiplayer/web#320]
Still waiting on this from November (#27) so creating a new issue and adding more to it.

### WEBSITE ADDITIONS

- add forum link to website menu
- add youtube and twitch links to footer of website. youtube.com/openmultiplayer twitch.tv/openmultiplayer

### REDIRECTS TO BE ADDED 
_feel free to give me the registrar login details and i'll do this myself_

> - **forum**.open.mp to burgershot.gg
> - open.mp/**forum** to burgershot.gg
>
> - **wiki**.open.mp to open.mp/docs
> - open.mp/**wiki** to open.mp/docs
> 
> - open.mp/**links** to http://linktr.ee/openmultiplayer

Those 4 are the most important. The rest below would just be nice additions if someone has time. 

- open.mp/**videos** to youtube.com/openmultiplayer
- open.mp/**youtube** to youtube.com/openmultiplayer

- open.mp/**live** to twitch.tv/openmultiplayer
- open.mp/**twitch** to twitch.tv/openmultiplayer
- open.mp/**stream** to twitch.tv/openmultiplayer

- open.mp/**twitter** to twitter.com/openmultiplayer
 
- open.mp/**instagram** to instagram.com/openmultiplayer

**Comment by Southclaws** [comment]
You can add more redirects by just editing that config file herehttps://github.com/openmultiplayer/web/commit/188cc0a59aef7efac34df2a08b5d7f37c048080a and they will go live a few minutes after being merged to main branch.

I haven't done the domains yet, I'll try and get that done soon!
